indexed-traversable/src/Data/Foldable/WithIndex.hs

```diff
@@ -96,28 +96,30 @@ itraverse_ f = void . getTraversed #. ifoldMap (Traversed #.. f)
 -- 'ifor_' ≡ 'flip' 'itraverse_'
 -- @
 --
--- When you don't need access to the index then 'for_' is more flexible in what it accepts.
+-- When you don't need access to the index then 'Data.Foldable.for_' is more flexible in what it accepts.
 --
 -- @
--- 'for_' a ≡ 'ifor_' a '.' 'const'
+-- 'Data.Foldable.for_' a ≡ 'ifor_' a '.' 'Data.Function.const'
 -- @
 ifor_ :: (FoldableWithIndex i t, Applicative f) => t a -> (i -> a -> f b) -> f ()
 ifor_ = flip itraverse_
 {-# INLINE ifor_ #-}
 
--- | Run monadic actions for each target of an 'IndexedFold' or 'Control.Lens.IndexedTraversal.IndexedTraversal' with access to the index,
+-- | Run monadic actions for each target of an 'ControlLens.Fold.IndexedFold' or 'Control.Lens.IndexedTraversal.IndexedTraversal' with
```

Comment:
Aren't these new links broken too, as there is no dependency on `lens` anywhere near?

Reply:
Yes you're right, there's no warning emitted by Haddock but the links are broken. I'll rewrite this to remove references to `lens`.

Reply:
I rewrote that documentation to remove the references to `lens`.

I also fixed another warning emitted by Haddock 2.30.0 by disambiguating type/constructor puns (previously I was testing with Haddock 2.29.2)